pages/user-interfaces.html

```diff
@@ -2,32 +2,21 @@
 <#include "incl/header.html">
 
 <div class="framework">
-<div class="frameworklabel">Editor-support packages</div>
+<div class="frameworklabel">Most popular user interfaces</div>
 <div class="frameworkcontent">
 
 <p>
-Coq projects can be developed with a variety of editors thanks to
+Coq projects can be developed with a variety of <strong>editors</strong> thanks to
 great support provided through user-contributed extensions.  Here, we
 list such support extensions that are actively used and maintained:
```

Comment:
The below can be hardly called "extensions" IMHO.

Reply:
Why not?

Reply:
As I wrote below it does refer to the particular way Coq support is implemented, so that's an unecessary technical detail to the user here. We could use instead "editor support for Coq"

Reply:
Ah, I now understand what you mean! I think this is because we understand "user-contributed extensions" differently. You think of extensions to Coq while I think of extensions to editors (and I don't know what Hugo had in mind). We could change this to say "user-contributed editor support packages" to clarify.

Reply:
Yes I prefer that. For example in the emacs world people don't talk about extensions but about "modes", so I was trying to find out a more common terminology.